app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -57,8 +57,13 @@ class ScanSimpleIdUpdate @Inject constructor(
   }
 
   private fun patientsFoundByOnlineLookup(completeMedicalRecords: List<CompleteMedicalRecord>): Next<ScanSimpleIdModel, ScanSimpleIdEffect> {
-    val patientId = completeMedicalRecords.first().patient.patientUuid
-    return dispatch(OpenPatientSummary(patientId))
+    val effect = if (completeMedicalRecords.isNotEmpty() && completeMedicalRecords.size > 1) {
```

Comment:
I prefer to move compound boolean logic like this into a separate local variable for better readability, i.e something like `atLeastOneMedicalRecordFound`.

Reply:
Also, the `.isNotEmpty()` check is redundant since the `size > 1` check should include that.